sdk/keyvault/azkeys/client.go

```diff
@@ -287,6 +282,16 @@ type CreateRSAKeyOptions struct {
 
 	// Application specific metadata in the form of key-value pairs.
 	Tags map[string]string `json:"tags,omitempty"`
+
+	// Elliptic curve name. For valid values, see KeyCurveName.
+	Curve *KeyCurveName `json:"crv,omitempty"`
+
+	// The attributes of a key managed by the key vault service.
+	KeyAttributes *KeyAttributes  `json:"attributes,omitempty"`
```

Comment:
We flatten this in .NET, but if that's not an ideal option for Go this should be `Properties *KeyProperties`, but perhaps that's better covered in your other PR you'd then need to merge into this topic branch eventually.